control-plane/subcommand/server-acl-init/create_or_update.go

```diff
@@ -61,6 +227,15 @@ func (c *Command) createACL(name, rules string, localToken bool, dc string, isPr
 		return err
 	}
 
+	// Check if the secret already exists, if so, we assume the ACL has already been
+	// created and return.
+	secretName := c.withPrefix(name + "-acl-token")
+	_, err = c.clientset.CoreV1().Secrets(c.flagK8sNamespace).Get(c.ctx, secretName, metav1.GetOptions{})
+	if err == nil {
+		c.log.Info(fmt.Sprintf("Secret %q already exists", secretName))
+		return nil
+	}
+
```

Comment:
It may be a strange diff but I think we already have this logic

Reply:
Ah you're right! This is unnecessary (duplicated right below) and was added by mistake when I did a large rebase, good catch!